docs/source/tutorial/package_a_project.md

```diff
@@ -145,6 +138,14 @@ main(
 
 This is equivalent to running `kedro run`, and you can provide all the parameters described by `kedro run --help`.
 
+Recipients of the `.egg` and `.whl` files need to have Python and `pip` on their machines, but do not need to have Kedro installed.
```

Comment:
I think we are removing all mention of `.egg` from the docs, right, @astrojuanlu ?